appconfiguration/client/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -72,10 +72,10 @@
     /**
      * Creates a builder that can configure options for the ConfigurationAsyncClient before creating an instance of it.
      *
-     * @return A new {@link ConfigurationAsyncClientBuilder} to create a ConfigurationAsyncClient from.
+     * @return A new {@link ConfigurationClientBuilder} to create a ConfigurationAsyncClient from.
      */
-    public static ConfigurationAsyncClientBuilder builder() {
-        return new ConfigurationAsyncClientBuilder();
+    public static ConfigurationClientBuilder builder() {
```

Comment:
Part of the plan is to drop these builder methods and to just have people new up the builder directly.